src/gewis/gewis.ts

```diff
@@ -356,6 +356,12 @@ export default class Gewis {
         Balance: {
           get: { all: star },
         },
+        Event : {
+          get: { all: star },
```

Comment:
I think that `GET /events` and `GET /events/:id` should be put in the RBAC of general SudoSOS users, because also non-BAC people should be able to fill in events planning sheets. Think of feuten and BHV'ers. However, then everyone can fetch complete planning sheets, so that is not a good solution either. For now, I think this is fine. Maybe we can add a new role to be able to fetch and fill in planning sheets? @JustSamuel